fastlane/lib/fastlane/actions/docs/upload_to_play_store.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="/img/actions/supply.png" width="250">
+  <img src="assets/supply.png" width="250">
```

Comment:
This will not work in the rendered state of the article - the image currently works on the docs site, so no need to change anything here.